hermes_eea/__init__.py

```diff
@@ -20,6 +21,24 @@
 INST_TO_TARGETNAME = {INST_NAME: INST_TARGETNAME}
 
 _package_directory = os.path.dirname(os.path.abspath(__file__))
+
 _data_directory = os.path.abspath(os.path.join(_package_directory, "data"))
 
+
 log.info(f"hermes_eea version: {__version__}")
+
+skeleton = str( os.path.join(_data_directory, "masterSkeletons", "hermes_eea_l0_00000000000000_v0.0.0.cdf") )
```

Comment:
The hermes container class should make it such that you do not need to use skeleton files.

Reply:
removed